cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilderTest.java

```diff
@@ -1944,37 +1944,37 @@ public void testOAuthOptions()  {
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_SCOPE, "messaging");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_AUDIENCE, "kafka");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_JWKS_ENDPOINT_URI, "http://jwks-endpoint");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_JWKS_REFRESH_SECONDS, "50");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_JWKS_EXPIRY_SECONDS, "160");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_JWKS_REFRESH_MIN_PAUSE_SECONDS, "5");
+        KafkaBrokerConfigurationBuilder.addBooleanOptionIfTrue(expectedOptions, ServerConfig.OAUTH_JWKS_IGNORE_KEY_USE, true);
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_INTROSPECTION_ENDPOINT_URI, "http://introspection-endpoint");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_USERINFO_ENDPOINT_URI, "http://userinfo-endpoint");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_USERNAME_CLAIM, "preferred_username");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_FALLBACK_USERNAME_CLAIM, "client_id");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_FALLBACK_USERNAME_PREFIX, "client-account-");
+        KafkaBrokerConfigurationBuilder.addBooleanOptionIfFalse(expectedOptions, ServerConfig.OAUTH_ACCESS_TOKEN_IS_JWT, false);
+        KafkaBrokerConfigurationBuilder.addBooleanOptionIfFalse(expectedOptions, ServerConfig.OAUTH_CHECK_ACCESS_TOKEN_TYPE, false);
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_VALID_TOKEN_TYPE, "access_token");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM, "");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_CONNECT_TIMEOUT_SECONDS, "30");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_READ_TIMEOUT_SECONDS, "60");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_HTTP_RETRIES, "2");
+        KafkaBrokerConfigurationBuilder.addOptionIfNotNull(expectedOptions, ServerConfig.OAUTH_HTTP_RETRY_PAUSE_MILLIS, "500");
+        KafkaBrokerConfigurationBuilder.addBooleanOptionIfTrue(expectedOptions, ServerConfig.OAUTH_ENABLE_METRICS, true);
+        KafkaBrokerConfigurationBuilder.addBooleanOptionIfFalse(expectedOptions, ServerConfig.OAUTH_FAIL_FAST, false);
```

Comment:
Is this change really necessary? In a way, you create the expected result using the same methods used to create the actual result which is wrong. The expected result should be as simple as possible -> so not calling any of these methods from the production code.

Reply:
I haven't noticed that, thanks Jakub